code/front/js/productFront.js

```diff
@@ -0,0 +1,148 @@
+/*
+** single product page
+*/
+
+//get _id of product from query parameters
+const queryString = window.location.search;
+console.log(queryString);
+let splitForId = queryString.slice(5);
+console.log(splitForId);
+
+const newUri = 'http://localhost:3000/api/products/' + splitForId;
```

Comment:
make it:
`const PRODUCT_URI  = 'http://localhost:3000/api/products/' + PRODUCT_ID;`